src/Symfony/Component/Console/Formatter/OutputFormatterInterface.php

```diff
@@ -26,7 +26,7 @@ public function setDecorated(bool $decorated);
     /**
      * Gets the decorated flag.
      *
-     * @return bool true if the output will decorate messages, false otherwise
```

Comment:
seems more useful than "Gets the decorated flag."

Reply:
updated